CommonControls/FileTypes/Animation/AnimationFile.cs

```diff
@@ -181,10 +181,18 @@ public int GetIdFromBoneName(string name)
             var boneInfo = Bones
                 .Where(x => string.Compare(x.Name, name, StringComparison.InvariantCultureIgnoreCase) == 0)
                 .FirstOrDefault();
-                            
+
             return (boneInfo == null) ? -1 : boneInfo.Id;
         }
 
+        /// <summary>
+        /// Safe method for looking up bones, as opposed to using the bone index directly
+        /// </summary>
+        public string GetBoneNameFromIndex(int boneIndex)
```

Comment:
this should not be added here. 
Feel free to add it as an extention method within the assetManagment project or as a helper function where used

Reply:
Yeah, oops,, I get it :)
I will remove both
`string GetBoneNameFromIndex(int boneIndex)`
`public int GetIdFromBoneName(string name)` 
from` AnimationFIle`
to `class AsseetMagement.AnimationProcessing.SkeletonHelper`